source/_integrations/switch_as_x.markdown

```diff
@@ -43,3 +43,16 @@ Switch entity:
 Type:
   description: The type of entity you want the switch to become.
 {% endconfiguration_basic %}
+
+## Alternatives
+Other ways to set a Switch as X include:
+ - [Helper](#helper)
+ - [Entity Properties](#entity-properties)
+
+### Helper
+{% my helpers title="Settings -> Devices & Services -> Helpers" %}
+![Helper example](/images/integrations/switch_as_x/Helper_Switch_as_X.png)
```

Comment:
I'm not sure about this, this is already provided by the steps above?

![image](https://user-images.githubusercontent.com/195327/182315806-ccda3fcf-37c4-4926-90b0-74912088bece.png)

Reply:
Sorry my bad. I will remove it.